### PR TITLE
ODS-5552 EdFi.Ods.Extensions.Homograph workflow added

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

File: .github/workflows/Packages - EdFi.Ods.Extensions.Homograph.yml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: Packages - EdFi.Ods.Extensions.Homograph
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
```
Comment (bptillman) [inline .github/workflows/Packages - EdFi.Ods.Extensions.Homograph.yml:12]: Looking at the exitsing build in TeamCity, this only got triggered on main builds or manually, so please remove this pull request trigger. If you need to keep it here to confirm everything works at first, that is fine. But once complete with any testing work, this should be removed.

File: .github/workflows/Packages - EdFi.Ods.Extensions.Homograph.yml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    branches:
+      - main      
+  workflow_dispatch:
+
+env:
+  INFORMATIONAL_VERSION: "6.1"
+  BUILD_INCREMENTER: "20"
+  CONFIGURATION: "Release"
+  CURRENT_BRANCH:  ${{ GITHUB.REF_NAME }}
```
Comment (bptillman) [inline .github/workflows/Packages - EdFi.Ods.Extensions.Homograph.yml:21]: Is this `CURRENT_BRANCH` env variable being used anywhere? I believe it was used in the previous workflows with the way we checked out the other repos before

**Review by bptillman — CHANGES_REQUESTED** [review]
Just some minor changes here, nothing major. Once they are changed then this can be approved.

File: .github/workflows/Packages - EdFi.Ods.Extensions.Homograph.yml (hunk 1 above)
Comment (semalaiappan) [reply .github/workflows/Packages - EdFi.Ods.Extensions.Homograph.yml]: Fixed it

File: .github/workflows/Packages - EdFi.Ods.Extensions.Homograph.yml (hunk 2 above)
Comment (semalaiappan) [reply .github/workflows/Packages - EdFi.Ods.Extensions.Homograph.yml]: removed it

**Review by bptillman — APPROVED** [review]

**Pull request merged**